src/demo/components/news/NewsCard.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+
+import './NewsFeed.css';
+
+export interface NewsItem {
+    date: Date;
+    headline: string;
+}
+
+export function NewsCard({item}: {item: NewsItem}) {
+    const {date, headline} = item;
+
+    return (
+        <li className="news-item w3-card">
+            <div className="news-item__date">
+                <span className="news-item__date__date">{date.toDateString()}</span>
```

Comment:
Is the `__` a convention? Not sure what this gets us

Reply:
It's based on [BEM](https://en.bem.info/methodology/quick-start/#nesting-1) although not strictly implemented. Didn't put much thought into it, happy to change to align better with the rest of the application

Reply:
Ah k, nah, sounds reasonable, happy for this to stay as is